car_item[] carlist      # 一张图检测出的车辆列表
armor_item[] armorlist  # 一张图检测出的装甲板列表